web/views.py

```
@@ -5575,8 +5588,9 @@ def award_badge(request):
         return JsonResponse({"success": False, "message": "Student not found"}, status=404)
     except Course.DoesNotExist:
         return JsonResponse({"success": False, "message": "Course not found"}, status=404)
-    except Exception:
-        return JsonResponse({"success": False, "message": "Error: award_badge"}, status=500)
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)